### PR TITLE
Asset packageName

### PR DESCRIPTION
**Pull request by d3uter**
Added posibility to specify asset package  to generate proper js links.

### CODE REVIEW

File: Twig/Extension/StfalconTinymceExtension.php [hunk 1]
```diff
@@ -163,7 +169,9 @@ public function tinymceInit($options = array())
             ),
             'include_jquery' => $config['include_jquery'],
             'tinymce_jquery' => $config['tinymce_jquery'],
+            'asset_package_name' => $asset_package_name,
             'base_url'       => $this->baseUrl
+
```
Comment (LogansUA) [inline Twig/Extension/StfalconTinymceExtension.php]: Please delete this blank line and align all elements of the array.
I think it was described in [PSR-2](http://www.php-fig.org/psr/psr-2/)

File: Twig/Extension/StfalconTinymceExtension.php [hunk 2]
```diff
@@ -84,13 +84,19 @@ public function tinymceInit($options = array())
         $config = array_merge_recursive($config, $options);
 
         $this->baseUrl = (!isset($config['base_url']) ? null : $config['base_url']);
+
+        #ASSET PACKAGE NAME
+        $asset_package_name = (!isset($config['asset_package_name']) ? null : $config['asset_package_name']);
```
Comment (LogansUA) [inline Twig/Extension/StfalconTinymceExtension.php]: Also please use camelCase for variable names

**Comment by LogansUA** [comment]
Hello. Please fix those bugs if you want us to approve this MR.

**Comment by d3uter** [comment]
Bugs fixed, thanks.

**Comment by fre5h** [comment]
@LogansUA looks good

**Comment by LogansUA** [comment]
@d3uter thanks for this PR.

**Pull request merged**